Habitica/src/com/habitrpg/android/habitica/helpers/MarkdownParser.java

```diff
@@ -0,0 +1,39 @@
+package com.habitrpg.android.habitica.helpers;
+
+import android.text.Html;
+
+import com.commonsware.cwac.anddown.AndDown;
+
+import net.sevenbase.emojicheatsheet.EmojiParser;
+
+/**
+ * Created by void on 12/24/15.
+ */
+public class MarkdownParser {
+
+    AndDown processor = new AndDown();
```

Comment:
the methods and this field can be made static, so that you don't need to create a MarkdownParser that often